### PR TITLE
Ready for review

### PR DESCRIPTION
**Pull request by michel0323**

### CODE REVIEW

**Comment by michel0323** [comment]
I welcome feedback. Thanks!

File: starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java [hunk 1]
```diff
@@ -1,52 +1,35 @@
...
-  private MongoTemplate mongoTemplate;
-  private InventoryDAO inventoryDAO;
-  private static final String NAME = "Amber";
-  private static final String PRODUCT_TYPE = "hops";
 
-  @Before
-  public void setup() {
-    this.inventoryDAO = new InventoryDAO(this.mongoTemplate);
-  }
+    @Mock private MongoTemplate mockMongoTemplate;
```
Comment (sofiac-developer) [inline starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java]: You shouldn't use a mock for mongo template. Utilize the mongo template set up in this test. 
Also you shouldn't remove anything from the test. Any lines here that are red should be added back. This file should only be adding to it, not removing

File: starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java [hunk 2]
```diff
@@ -1,52 +1,35 @@
...
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.boot.test.autoconfigure.data.mongo.DataMongoTest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
```
Comment (sofiac-developer) [inline starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java]: You shouldn't need Mockito for this test.

File: starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java [hunk 3]
```diff
@@ -49,10 +49,13 @@ public List<Inventory> findAll() {
    * Save Inventory.
    * @param inventory Inventory to Save/Update.
    * @return Created/Updated Inventory.
+   * 
    */
   public Inventory create(Inventory inventory) {
-    // TODO
-    return null;
+    //Modified by Michel T. on 08/25/22
+    inventory.setId(null);
```
Comment (sofiac-developer) [inline starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java:56]: Looks good

**Review by sofiac-developer — CHANGES_REQUESTED** [review]

File: starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java [hunk 4]
```diff
@@ -14,39 +16,53 @@
...
+	}
+	
+	@Test
+    public void create() {
+        Inventory inventory = new Inventory();
+        inventory.setName(NAME);
+		inventory.setProductType(PRODUCT_TYPE);
+        Inventory actual = inventoryDAO.create(inventory);
+        assertNotNull(actual);
+        assertNotNull(actual.getName());
```
Comment (jinlin0) [inline starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java:66]: add an assert for the inventory id

**Pull request merged**